frontend/src/views/EditStock.vue

```diff
@@ -0,0 +1,70 @@
+<script setup>
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { useProductStore } from '../store/productStore.js'
+import { useRoute } from 'vue-router'
+import { ref } from 'vue'
+import router from '../router/router.js'
+
+const route = useRoute()
+const productStore = useProductStore()
+
+const productId = Number(route.params.id)
+const productFound = await productStore.get(productId)
+const amount = ref(0)
+
+async function update () {
+    const { id, ...data } = productFound
+    await productStore.update(data, id)
+}
+</script>
+
+<template>
+  <div class="popup-container box text-center bg-white justify-content-center">
+    <router-link type="button" class="btn float-start hover-darken" :to="`/products/${productId}`">
+      <font-awesome-icon :icon="['fas', 'circle-info']"/>
```

Comment:
Op de pagina is het een beetje onduidelijk dat deze knop naar het product toe gaat. Mischien hier de stapel van dozen of doos van maken? Dan is het duidelijk dat het over het product gaat.